src/widgets/MeterPanel.cpp

```diff
@@ -671,65 +716,16 @@ void MeterPanel::OnPaint(wxPaintEvent & WXUNUSED(event))
    }
 #endif
 
-   // Let the user know they can click to start monitoring
-   if( mIsInput && !mActive )
+   if (mStyle != MixerTrackCluster)
```

Comment:
Now I see it.  All the painting of the slider is skipped for mixer tracks.

Maybe then mSlider should remain null in case of mixer board clusters.  Avoid making a useless LWSlider.